### DISCUSSION TITLE
Adding Student Email to Personal Account After Deleting Student Account

### DISCUSSION THREAD

**Discussion by ghost** [community/community#143888] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,

When I first enrolled in college, I created a GitHub account using my student email, thinking it was the only way to access GitHub's student benefits. Now that I realize I can create a personal account and add my student email to it, I’m concerned about my ongoing projects in my student account.

I’ve already used the student benefits. If I delete my student account, will I be able to add my student email to my personal account without losing access to my repositories or benefits? And what about all my ongoing projects?

**Comment by servusdei2018** [comment]
You can have multiple emails linked to your GitHub account, and it's easy to switch which one is the primary one:

- [Changing your primary email address](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/changing-your-primary-email-address)
- [Adding an email address to your GitHub account](https://docs.github.com/en/enterprise-server@3.10/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/adding-an-email-address-to-your-github-account)

In addition, you can transfer repositories from one account to another:

- [Transferring a repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository)

Through a combination of the two you should be able to merge both accounts into one, and keep all of your repositories and data intact.

I would suggest, since your educational account is older, transferring all repositories to that from your new account, closing your new account, linking your new email to your educational account then making that your primary email; in that order.

**Reply by ghost** [reply]
So I should delete my personal account so that its email will be free; add this email to my student account; change the email to my primary

With all that, the primary email means it will be the one that I will use to log in and share with others, not my old student one?

Not quite sure I understand what this primary email means...

**Comment by servusdei2018** [comment]
Yes, that is correct.
